src/Tizen.NUI/src/internal/LinearConstrainer.cs

```diff
@@ -65,107 +55,17 @@ protected override void Dispose(DisposeTypes type)
             base.Dispose(type);
         }
 
-        /// <since_tizen> 3 </since_tizen>
-        public class Property : global::System.IDisposable
+        internal class Property
```

Comment:
please make this HiddenAPI (not internal) ; please add [EditorBrowsable], [Obsolete] attributes like others done.